influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -358,17 +373,44 @@ pub(crate) fn parse_validate_and_update_schema(
     schema: &DatabaseSchema,
     partitioner: &Partitioner,
     default_time: i64,
+    accept_partial: bool,
+    precision: Precision,
 ) -> Result<ValidationResult> {
     let mut lines = vec![];
+    let mut errors = vec![];
+    let mut valid_lines = vec![];
+    let mut lp_lines = lp.lines();
+
     for (line_idx, maybe_line) in parse_lines(lp).enumerate() {
```

Comment:
The `unwraps` in this loop are safe, may be worth putting in a comment indicating that.